src/test/scala/karazin/scala/users/group/week2/homework/HomeworkSpecification.scala

```diff
@@ -50,28 +50,29 @@ object HomeworkSpecification extends Properties("Homework"):
   }
 
   property("negation") = forAll { (rational: Rational) =>
-    ???
+    -rational == Rational(-rational.numer, rational.denom)
```

Comment:
The test framework usually assume that the first component is an actual result, and the second one is an expected one. Assuming that the framework shows the differences between objects. So, it's better to follow this order: 1) what to check; 2) expected result.

Reply:
In this example i am testing the left negation with the right expected result as it follows